### ISSUE TITLE
Scrolling not working with keyboard in AutoComplete

### ISSUE THREAD

**Issue by Lindsay-Mathieson** [microsoft/fast-blazor#2220]
# 🐛 Bug Report

When an auto complete has many items in its drop down, which require scrolling to access, using the keyboard down arrow to navigate down the list causes the selection bar to vanish below the list viewport. Scrolling with the mouse works fine

## 💻 Repro or Code Sample

The Many Items sample from the demo site

https://www.fluentui-blazor.net/Autocomplete#manyitems

## 🤔 Expected Behavior

After opening the autocomplete dropdown, using the down arrow to move down the list should cause scrolling when the selection bar hits the bottom of the dropdown pane

## 😯 Current Behavior

The selected row moves out of sight. Using the Up arrow brings it back into sight.

## 🔦 Context

I'm developing a data entry page for customers in a WPF Blazor app which needs to be keyboard accessible.

## 🌍 Your Environment
* Windows desktop, Webbrowser
* WebView2 in a a windows WPF App, Edge (Windows)
* .NET 8, FluentUI 4.8, 4.7.2

**Comment by dvoituron** [comment]
Except for reducing the number of filtered values or adding a style to increase the height of the popup, we don't yet have a solution.
```css
.fluent-autocomplete-multiselect div[role=listbox] {
  max-height: 400px;
}
```

**Comment by Lindsay-Mathieson** [comment]
> Except for reducing the number of filtered values or adding a style to increase the height of the popup, we don't yet have a solution.

Ah, thanks, good to know. It's not a huge issue for us as they can use the mouse or as you say reduce the items. I'll look into alternative dropdowns, fluentui certainly does have many of them!

**Comment by dvoituron** [comment]
I've found a solution to this problem. It will be included in the next version.
See #2221

**Comment by Lindsay-Mathieson** [comment]
> I've found a solution to this problem. It will be included in the next version. See #2221

Brilliant, thankyou

**Comment by Lindsay-Mathieson** [comment]
Can confirm this is fixed in 4.9, thank you so much!
